src/transaction/sign.ts

```diff
@@ -1,3 +1,4 @@
+import * as isEqual from '../common/js/lodash.isequal'
 import * as utils from './utils'
 import keypairs = require('ripple-keypairs')
 import binary = require('ripple-binary-codec')
```

Comment:
optional: Consider rebasing this import to binaryCodec because it makes the code below easier to read IMO:

```javascript
const decoded = binary.decode(serialized)
```
vs
```javascript
const decoded = binaryCodec.decode(serialized)
```